src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -447,7 +454,6 @@ public Size Measure(Size available, Size visibleSize)
 				{
 					// Size : Note we set both Min and Max to match the UWP behavior which alter only those properties
 					child.MinWidth = available.Width;
-					child.MinHeight = available.Height;
```

Comment:
You should add a comment to explain why this line is "missing", it might be confusing for future dev.

Reply:
It's in the associated comments on the IsPopupFullscreen, but I can extend the comment above.